semcore/popper/src/Popper.jsx

```diff
@@ -183,6 +185,13 @@ class Popper extends Component {
       name: 'computeStyles',
       options: { gpuAcceleration: false },
     });
+    modifiersOptions.push({
+      name: 'cursorAnchoring',
+      options: {
+        cursorAnchoring: this.asProps.cursorAnchoring,
+        mouseEnterCursorPositionRef: this.mouseEnterCursorPositionRef,
+      },
+    });
```

Comment:
maybe we should push this modifier only if we have `this.asProps.cursorAnchoring` ?